Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -165,6 +165,10 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         var cmd = [resources.swiftCompilerPath.asString]
         cmd += ["--driver-mode=swift"]
         cmd += verbosity.ccArgs
+    #if CYGWIN
+        cmd += ["-D", "CYGWIN"]
```

Comment:
This code is loading the *manifest* though, and we don't want users to conditionalize their manifests in this way. Since it isn't currently required, I recommend we avoid it here.